modules/custom/openy_gtranslate/js/openy_gtranslate.js

```diff
@@ -0,0 +1,25 @@
+(function ($) {
+
+  Drupal.behaviors.openy_gtranslate = {
+    attach: function (context, settings) {
+      var langSelect = $('.goog-te-menu-frame').first();
+
+      $('nav .navbar-nav li.language > a').on('click', function (e, context) {
+        e.preventDefault();
+        langSelect.css('left', '0px');
+        langSelect.css('top', '150px');
```

Comment:
Could you operate by classes and avoid of adding styles int the code?
MVC pattern is really important here